docker-compose.yml

```
@@ -1,36 +1,47 @@
 version: "3"
 
 services:
-  db:
+  mongo:
     container_name: mongo
     image: mongo
+    ports:
+      - "27017:27017"
 
-  server:
-    container_name: server
-    build: server/
-    restart: unless-stopped
-    command: python ./app.py
+  redis:
+    container_name: redis
+    image: redis
     ports:
-      - "5000:80"
-    environment:
-      FLASK_PORT: 80
-      DB_URI: "mongodb://db:27017/db"
-      WORKER_URI: "http://worker/"
+      - "6379:6379"
+
+  celery:
```

```suggestion
  celery:
    container_name: celery
```
Чтобы название контейнера были единообразными.

---

✓